### PR TITLE
Fix release condition

### PR DESCRIPTION
**Pull request by tomjebo**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -33,6 +24,18 @@ jobs:
         - name: Setup .NET Core
           uses: actions/setup-dotnet@v3
 
+        - name: Approve
+          uses: trstringer/manual-approval@v1
+          with:
+              secret: ${{ github.TOKEN }}
+              approvers: mikeebowen,tomjebo,twsouthwick,alfredhellstern
```
Comment (twsouthwick) [inline .github/workflows/release.yml:31]: we should set up a github team for this

**Review by twsouthwick — DISMISSED** [review]

**Comment by twsouthwick** [comment]
My comment is not a blocker - but we should create an issue to track it

File: .github/workflows/release.yml (hunk 1 above)
Comment (tomjebo) [reply .github/workflows/release.yml:31]: we can look into that.

**Comment by tomjebo** [comment]
> My comment is not a blocker - but we should create an issue to track it

that's what I assumed. I looked into it initially but looks like we don't have the rights to create a team in dotnet org on Github. So we might have to ask someone to do that for us unless there's another way to create a team in Github.

**Pull request merged**